xml_inscriptions/transcribed/CA.Malibu.JPGM.L.80.AD.10.2.xml

```diff
@@ -178,14 +178,25 @@
              
          <div type="translation">
             
-            <p>Work from the Sulpicianae brickyards of Marcus Vinicius Pantagatus, with Quintus
+            <p>Sulpician work of Marcus Vinicius Pantagatus, with Quintus
                Articuleius Paetinus (and) Lucius Venuleius Apronianus (being) consuls</p>
             
             <p/>
             
          </div>
          <div type="commentary">
-            <p>Rectangular with hollow letters (Forma 111)</p>
+            
+            <p>
+               <lb n="1"/> Form: Rectangular with hollow letters
```

Comment:
Typically you should put individual `<p>` elements for each new line in the commentary, as `<lb>` doesn't always work here (I'm also not sure it's technically correct to call these individual line beginnings in the same paragraph).